### ISSUE TITLE
Close all open text editors causes crash

### ISSUE THREAD

**Issue by fire** [godotengine/godot#49785]
### Godot version

d696bd1

### System information

popos 20.04

### Issue description

Editor crashes when I use the right menu close all scripts.

### Steps to reproduce

1. use the right menu close all scripts.
2. crash

### Minimal reproduction project

Any project will crash.

### Other

https://github.com/godotengine/godot/blob/d696bd17ed1b7019108c8a573712f00ac2d735a3/editor/code_editor.cpp#L593-L595

**Comment by Calinou** [comment]
I can confirm this on `master` commit https://github.com/godotengine/godot/commit/9da7f0690046746b2f1430717898abd6377d003a.

Backtrace:

```
[1] /lib64/libc.so.6(+0x3d320) [0x7f21d2fad320] (??:0)
[2] Object::get_class_name() const (/home/hugo/Documents/Git/godotengine/godot/./core/object/object.h:694)
[3] Object::_disconnect(StringName const&, Callable const&, bool) (/home/hugo/Documents/Git/godotengine/godot/core/object/object.cpp:1385)
[4] Object::disconnect(StringName const&, Callable const&) (/home/hugo/Documents/Git/godotengine/godot/core/object/object.cpp:1375)
[5] FindReplaceBar::set_text_edit(CodeTextEditor*) (/home/hugo/Documents/Git/godotengine/godot/editor/code_editor.cpp:594)
[6] CodeTextEditor::set_find_replace_bar(FindReplaceBar*) (/home/hugo/Documents/Git/godotengine/godot/editor/code_editor.cpp:962)
[7] ScriptTextEditor::set_find_replace_bar(FindReplaceBar*) (/home/hugo/Documents/Git/godotengine/godot/editor/plugins/script_text_editor.cpp:1357)
[8] ScriptEditor::ensure_select_current() (/home/hugo/Documents/Git/godotengine/godot/editor/plugins/script_editor_plugin.cpp:1643)
[9] ScriptEditor::_tab_changed(int) (/home/hugo/Documents/Git/godotengine/godot/editor/plugins/script_editor_plugin.cpp:1465)
[10] void call_with_variant_args_helper<ScriptEditor, int, 0ul>(ScriptEditor*, void (ScriptEditor::*)(int), Variant const**, Callable::CallError&, IndexSequence<0ul>) (/home/hugo/Documents/Git/godotengine/godot/./core/variant/binder_common.h:214)
[11] void call_with_variant_args<ScriptEditor, int>(ScriptEditor*, void (ScriptEditor::*)(int), Variant const**, int, Callable::CallError&) (/home/hugo/Documents/Git/godotengine/godot/./core/variant/binder_common.h:324)
[12] CallableCustomMethodPointer<ScriptEditor, int>::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/hugo/Documents/Git/godotengine/godot/./core/object/callable_method_pointer.h:97)
[13] Callable::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/hugo/Documents/Git/godotengine/godot/core/variant/callable.cpp:51)
[14] Object::emit_signal(StringName const&, Variant const**, int) (/home/hugo/Documents/Git/godotengine/godot/core/object/object.cpp:1083)
[15] Object::emit_signal(StringName const&, Variant const&, Variant const&, Variant const&, Variant const&, Variant const&) (/home/hugo/Documents/Git/godotengine/godot/core/object/object.cpp:1138)
[16] TabContainer::set_current_tab(int) (/home/hugo/Documents/Git/godotengine/godot/scene/gui/tab_container.cpp:763)
[17] ScriptEditor::_close_tab(int, bool, bool) (/home/hugo/Documents/Git/godotengine/godot/editor/plugins/script_editor_plugin.cpp:753)
[18] ScriptEditor::_close_current_tab(bool) (/home/hugo/Documents/Git/godotengine/godot/editor/plugins/script_editor_plugin.cpp:767)
[19] ScriptEditor::_close_all_tabs() (/home/hugo/Documents/Git/godotengine/godot/editor/plugins/script_editor_plugin.cpp:818)
[20] ScriptEditor::_menu_option(int) (/home/hugo/Documents/Git/godotengine/godot/editor/plugins/script_editor_plugin.cpp:1359)
[21] void call_with_variant_args_helper<ScriptEditor, int, 0ul>(ScriptEditor*, void (ScriptEditor::*)(int), Variant const**, Callable::CallError&, IndexSequence<0ul>) (/home/hugo/Documents/Git/godotengine/godot/./core/variant/binder_common.h:214)
[22] void call_with_variant_args<ScriptEditor, int>(ScriptEditor*, void (ScriptEditor::*)(int), Variant const**, int, Callable::CallError&) (/home/hugo/Documents/Git/godotengine/godot/./core/variant/binder_common.h:324)
[23] CallableCustomMethodPointer<ScriptEditor, int>::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/hugo/Documents/Git/godotengine/godot/./core/object/callable_method_pointer.h:97)
[24] Callable::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/hugo/Documents/Git/godotengine/godot/core/variant/callable.cpp:51)
[25] Object::emit_signal(StringName const&, Variant const**, int) (/home/hugo/Documents/Git/godotengine/godot/core/object/object.cpp:1083)
[26] Object::emit_signal(StringName const&, Variant const&, Variant const&, Variant const&, Variant const&, Variant const&) (/home/hugo/Documents/Git/godotengine/godot/core/object/object.cpp:1138)
[27] PopupMenu::activate_item(int) (/home/hugo/Documents/Git/godotengine/godot/scene/gui/popup_menu.cpp:1379)
[28] PopupMenu::_gui_input(Ref<InputEvent> const&) (/home/hugo/Documents/Git/godotengine/godot/scene/gui/popup_menu.cpp:392)
[29] void call_with_variant_args_helper<PopupMenu, Ref<InputEvent> const&, 0ul>(PopupMenu*, void (PopupMenu::*)(Ref<InputEvent> const&), Variant const**, Callable::CallError&, IndexSequence<0ul>) (/home/hugo/Documents/Git/godotengine/godot/./core/variant/binder_common.h:209)
[30] void call_with_variant_args<PopupMenu, Ref<InputEvent> const&>(PopupMenu*, void (PopupMenu::*)(Ref<InputEvent> const&), Variant const**, int, Callable::CallError&) (/home/hugo/Documents/Git/godotengine/godot/./core/variant/binder_common.h:324)
[31] CallableCustomMethodPointer<PopupMenu, Ref<InputEvent> const&>::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/hugo/Documents/Git/godotengine/godot/./core/object/callable_method_pointer.h:97)
[32] Callable::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/hugo/Documents/Git/godotengine/godot/core/variant/callable.cpp:51)
[33] Object::emit_signal(StringName const&, Variant const**, int) (/home/hugo/Documents/Git/godotengine/godot/core/object/object.cpp:1083)
[34] Object::emit_signal(StringName const&, Variant const&, Variant const&, Variant const&, Variant const&, Variant const&) (/home/hugo/Documents/Git/godotengine/godot/core/object/object.cpp:1138)
[35] Window::_window_input(Ref<InputEvent> const&) (/home/hugo/Documents/Git/godotengine/godot/scene/main/window.cpp:913)
[36] void call_with_variant_args_helper<Window, Ref<InputEvent> const&, 0ul>(Window*, void (Window::*)(Ref<InputEvent> const&), Variant const**, Callable::CallError&, IndexSequence<0ul>) (/home/hugo/Documents/Git/godotengine/godot/./core/variant/binder_common.h:209)
[37] void call_with_variant_args<Window, Ref<InputEvent> const&>(Window*, void (Window::*)(Ref<InputEvent> const&), Variant const**, int, Callable::CallError&) (/home/hugo/Documents/Git/godotengine/godot/./core/variant/binder_common.h:324)
[38] CallableCustomMethodPointer<Window, Ref<InputEvent> const&>::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/hugo/Documents/Git/godotengine/godot/./core/object/callable_method_pointer.h:97)
[39] Callable::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/hugo/Documents/Git/godotengine/godot/core/variant/callable.cpp:51)
[40] DisplayServerX11::_dispatch_input_event(Ref<InputEvent> const&) (/home/hugo/Documents/Git/godotengine/godot/platform/linuxbsd/display_server_x11.cpp:2639)
[41] DisplayServerX11::_dispatch_input_events(Ref<InputEvent> const&) (/home/hugo/Documents/Git/godotengine/godot/platform/linuxbsd/display_server_x11.cpp:2623)
[42] Input::_parse_input_event_impl(Ref<InputEvent> const&, bool) (/home/hugo/Documents/Git/godotengine/godot/core/input/input.cpp:643)
[43] Input::parse_input_event(Ref<InputEvent> const&) (/home/hugo/Documents/Git/godotengine/godot/core/input/input.cpp:466)
[44] Input::flush_accumulated_events() (/home/hugo/Documents/Git/godotengine/godot/core/input/input.cpp:858)
[45] DisplayServerX11::process_events() (/home/hugo/Documents/Git/godotengine/godot/platform/linuxbsd/display_server_x11.cpp:3473)
[46] OS_LinuxBSD::run() (/home/hugo/Documents/Git/godotengine/godot/platform/linuxbsd/os_linuxbsd.cpp:274)
[47] bin/godot.linuxbsd.tools.64.llvm(main+0x1cb) [0x489757b] (/home/hugo/Documents/Git/godotengine/godot/platform/linuxbsd/godot_linuxbsd.cpp:58)
[48] /lib64/libc.so.6(__libc_start_main+0xd5) [0x7f21d2f97b75] (??:0)
[49] bin/godot.linuxbsd.tools.64.llvm(_start+0x2e) [0x48972ee] (??:?)
-- END OF BACKTRACE --
[1]    7754 IOT instruction (core dumped)  bin/godot.linuxbsd.tools.64.llvm 
```

**Comment by KoBeWi** [comment]
I can't reproduce, but might be fixed by #49650 (stack trace is similar). Can you check?
